### DISCUSSION TITLE
Adding exit button on Android

### DISCUSSION THREAD

**Discussion by AxhyVN** [ppy/osu#30296] [Ideas] [answered]
I found a few discussions about this from like ~5 years ago but most of them weren't responded. And now (October 15th 2024), there still isn't any exit button ingame. I think it's just a small thing and it wouldn't take you long to add so I hope it will be added in the near future. I want to listen to "See you next time" on mobile for so long. (P/s: I hope the osu! team will keep working and taking feedback from community for the best version of lazer)

**Comment by Joehuu** [comment, marked as answer]
See past discussion in https://github.com/ppy/osu/discussions/16546 for answer.

**Reply by AxhyVN** [reply]
Thank you
